src/modules/adhs/test.json

```diff
@@ -0,0 +1,51 @@
+{
+  "modules": [
+    "adhs"
+  ],
+  "setup": [
+    "mkdir -p /etc/azure-device-health-services/",
+    "touch /etc/azure-device-health-services/config.toml",
+    "cp /etc/azure-device-health-services/config.toml config.toml"
+  ],
+  "teardown": [
+    "cp config.toml /etc/azure-device-health-services/config.toml",
+    "rm -f config.toml"
+  ],
+  "steps": [
```

Comment:
This does not appear right to have these test.json in the source code for each module. ModulesTest is a functional test that needs to test module binaries against test recipes. When this test is executed the OSConfig source code may not be present/available.

The contract for ModulesTest is:

1. Its binary must test module SO binaries
2. Against test recipes